# The goal
string gripper_link_name
string tool_link_name
string utility_link_name
string constraints
---
# The result
float64 distance
---
# The feedback
float64 distance

